test/support/api/workspace-patients.js

```diff
@@ -1,12 +1,16 @@
 import _ from 'underscore';
-import { getResource } from 'helpers/json-api';
+import { getResource, mergeJsonApi } from 'helpers/json-api';
 
 import fxWorkspacePatients from 'fixtures/collections/workspace-patients';
 
 const TYPE = 'workspace-patients';
 
-export function getWorkspacePatient() {
-  return getResource(_.sample(fxWorkspacePatients), TYPE);
+export function getWorkspacePatient(data) {
+  const resource = getResource(_.sample(fxWorkspacePatients), TYPE);
+
+  if (!data) return resource;
+
+  return mergeJsonApi(resource, data);
```

Comment:
More I need to add to this? I feel like I'm missing something. For example, some of these have a `VALID: ` portion added to the `mergeJsonApi()` parameters.

Reply:
that's for setting the possible valid keys of a relationship.  turning on debugging will show anywhere where a relationship is incorrectly used in the tests if the `VALID`s are all sett

Reply:
that said.. there don't look to be relationships here